### DISCUSSION TITLE
Self hosted

### DISCUSSION THREAD

**Discussion by rithythul** [tryethernal/ethernal#375] [General]
Hello there, thanks so much for open source Ethernal. We are small team base in Cambodia. Very happy to see that it has been open source. We would like to self host the instance for our chain but the https://doc.tryethernal.com/getting-started/installation is not found. Could you help point us to the place where we can learn how to run the app, please? 

BR, 

Rithy

**Comment by antoinedc** [comment]
Hi,
Sorry for the delayed answer! Self hosted instructions are still being rewritten and the page had been taken down because it was obsolete, due to a lot of changes. Hopefully, I'll able to finish this soon, but I don't have a timeline yet for you unfortunately.

**Comment by antoinedc** [comment]
Hey @rithythul,

We just released a big update to make it a lot easier to self host Ethernal, if you are still interested. All instructions are in the Readme file :)
